mithril-stm/src/lib.rs

```diff
@@ -117,6 +117,7 @@ mod protocol;
 mod signature_scheme;
 
 pub use protocol::*;
+pub use signature_scheme::BlsSignatureError;
```

Comment:
This should be gated behind the `benchmark-internals` feature as we don't want to have these internal types exposed on the public API.